modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -4692,6 +4697,7 @@ created-oldest-first=Created (oldest first)
 created-on=Created on {0}
 created-x-ago=Created {0} ago.
 creating-content=Creating content...
+creating-first-unit-of-measure-for-this-sku=By creating the first Unit of Measure for this SKU, all existing SKU Stock and Pricing (if any) will be mapped to the first Unit of Measure created. If you wish to continue selling by this SKU by the unit, it is recommended to create this Unit of Measure first. Otherwise you can continue and any existing stock and pricing (if any) will be mapped to this Unit of Measure
```

Comment:
can you please review this piece of message "selling by this SKU by the unit" it looks wrong to me, also the final part looks confusing to me